playbooks/container-registry/quay-enterprise.yml

```diff
@@ -53,32 +53,35 @@
           lb_ssl_enabled: True
       lb_backend_template: "{{ playbook_dir }}/templates/haproxy_backend.cfg.j2"
 
-- name: Install Database
+- name: Install Quay Database
   hosts: database
   tasks:
     - name: Install MySQL
```

Comment:
To be consistent with the below, can we change to `Install and Configure MySQL for Quay`?